src/main/resources/static/js/script.js

```diff
@@ -152,17 +151,26 @@ async function shoot(x, type) {
     console.log(shootMessage);
     switch (shotResult) {
         case 'MISS': {
+            shotMastSound.load()
```

Comment:
nie potrzebne w sumie

Reply:
Jeżeli poprzedni strzał był trafiony, to odpaliło to shotMastSound. Jeżeli odpowiedź na kolejny - tym razem nietrafiony - strzał zdążyła przyjść zanim poprzedni shotMastSound przestał grać, to do tego kolejnego - nietrafionego - strzału nie będzie już grać muzyka. Na pewno usunąć?

Reply:
będzie grać a przynajmniej powinna bo to są dwa różne obiekty. A dobrze jest dać wybrzmieć dźwiękom